doc/en/components/grids/_shared/cell-editing.md

```diff
@@ -616,10 +708,39 @@ The result of the above validation being applied to our `{ComponentName}` can be
 
 `sample="/{ComponentSample}/editing-events", height="650", alt="{Platform} {ComponentTitle} Editing Event Example"`
 
-<!-- Angular -->
-
 ## Styling
 
+<!-- WebComponents, Blazor -->
+
+In addition to the predifined themes, the grid could be further customized by setting some of the available [CSS Properties](theming.md).
+In case you would like to change some of the colors, you need to set a class for the grid first:
+
+```ts
+<igc-grid class="grid">
+```
+
+```razor
+<IgbGrid Class="grid"></IgbGrid>
+```
+
+Then set the related CSS properties for that class:
+
+```css
+.grid {
+    igx-grid__td--editing-background: #57A0D2;
```

Comment:
Snippet is incorrect. Prefix should be `--igx-grid` not igx-grid__td.
Preferably make it match the vars used in the sample:
```
    --igx-grid-edit-mode-color: #FFA500;
    --igx-grid-cell-active-border-color: #FFA500;
    --igx-grid-cell-editing-background: #add8e6;
```